src/chart/Pie.js

```diff
@@ -37,6 +33,9 @@
 
     Pie.prototype.publish("paletteID", "default", "set", "Palette ID", Pie.prototype._palette.switch(),{tags:["Basic","Shared"]});
     Pie.prototype.publish("useClonedPalette", false, "boolean", "Enable or disable using a cloned palette",null,{tags:["Intermediate","Shared"]});
+    Pie.prototype.publish("outerText", false, "boolean", "Sets label position inside or outside chart",null,{tags:["Basic"]});
+    Pie.prototype.publish("radius", 100, "number", "Sets radius in pixels of pie chart",null,{tags:["Basic"]});
+    Pie.prototype.publish("innerRadius", 0, "number", "Sets radius in pixels of center hole in pie chart (donut)",null,{tags:["Basic"]});
```

Comment:
publishing radius and innerRadius is a bit misleading as they just get overridden when the Pie chart gets sized?  I think innerRadius should be %?